app/medInria/medHomepageArea.cpp

```diff
@@ -284,33 +314,9 @@ medHomepageArea::medHomepageArea ( QWidget * parent ) : QWidget ( parent ), d (
     d->stackedWidget->addWidget(d->aboutWidget);
     d->stackedWidget->addWidget(d->pluginWidget);
     d->stackedWidget->addWidget(d->settingsWidget);
+    d->stackedWidget->addWidget(d->logWidget);
     d->stackedWidget->setCurrentIndex(0);//d->infoWidget
     d->stackedWidget->setSizePolicy(QSizePolicy::Maximum,QSizePolicy::Maximum);
-
-    //Setup homepage animations
```

Comment:
weeeellll... it was not the most beautiful animation of the world ^^'. I agree that sometimes animations, shadows, etc are interesting, but this kind... it reminds me the huge animations on powerpoint presentations... you see? ^^'